press-kit/index.md

```diff
@@ -120,7 +120,7 @@ meta_descr: meta_descr.presskit
   <div class="info-block">
     <h2>{% t press-kit.pressdoc %}</h2>
     <div>
-        <h3><a href="https://www.monerooutreach.org/quick-facts.html">{% t press-kit.quickfacts %}</a></h3>
+        <h3><a href="Monero Quick Facts_May_2019.pdf">{% t press-kit.quickfacts %}</a></h3>

```

Comment:
```suggestion
        <h3><a href="Monero_Quick_Facts-May_2019.pdf">{% t press-kit.quickfacts %}</a></h3>
```

perhaps better naming the file without spaces

Reply:
just following what our forefathers named files like https://github.com/monero-project/monero-site/blob/master/press-kit/Monero%20Quick%20Facts_May2018.pdf

Reply:
Up to you how to proceed. I think it looks cleaner (and easier to use). A mix of spaces and underscores looks funny to me